ros2bag/ros2bag/verb/play.py

```diff
@@ -37,131 +37,135 @@ def positive_float(arg: str) -> float:
     return value
 
 
+def add_player_arguments(parser: ArgumentParser) -> None:
+    add_standard_reader_args(parser)
+    parser.add_argument(
+        '--read-ahead-queue-size', type=int, default=1000,
+        help='size of message queue rosbag tries to hold in memory to help deterministic '
+             'playback. Larger size will result in larger memory needs but might prevent '
+             'delay of message playback.')
+    parser.add_argument(
+        '-r', '--rate', type=check_positive_float, default=1.0,
+        help='rate at which to play back messages. Valid range > 0.0.')
+    parser.add_argument(
+        '--topics', type=lambda s: list(s.split(' ')), default=[], metavar='topic',
+        help='Space-delimited list of topics to play.')
+    parser.add_argument(
+        '--services', type=lambda s: list(s.split(' ')), default=[], metavar='service',
+        help='Space-delimited list of services to play.')
```

Comment:
As fujitatomoya mentioned, "and quoted" should also be added to the description.